modules/js-and-css/speculation-rules/readme.txt

```diff
@@ -0,0 +1,84 @@
+=== Speculation Rules ===
+
+Contributors:      wordpressdotorg
+Requires at least: 6.3
+Tested up to:      6.4
+Requires PHP:      7.0
+Stable tag:        1.0.0
+License:           GPLv2 or later
+License URI:       https://www.gnu.org/licenses/gpl-2.0.html
+Tags:              performance, javascript, speculation rules, prerender, prefetch
+
+Uses the Speculation Rules API to prerender linked URLs upon hover.
```

Comment:
It says "prerender" here but "prefetch or prerender" (or words to that effect) everywhere else. Should we be consistent?

Reply:
The "prerender on hover" is the default behavior, that's why I chose to go with that here.

Reply:
Could we add "by default (with configuration options)" then to make that super obvious?

Reply:
Adding "by default" makes sense to me, will update. I think the "with configuration options" bit is too long, and is explained below, so IMO that's sufficient.